cypress/integration/cucumber/step_definitions/oclsubscriptionmodule/index.js

```diff
@@ -0,0 +1,108 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+const userApiToken = 'e6415c890158e06c36f7f52eca335bdf7204d50f'; //TODO: Fetch from the API and load directly
+const sUrl = 'https://api.staging.openconceptlab.org/users/openmrs/collections/TCM/1/';
+
+const fetchData = (url) => fetch(url)
+  .then((response) => {
+    return response.json();
+  }).then((data) => {
+    return data;
+  });
+
+Given('the user goes to the Dictionary Manager App', () => {
+  cy.visit('https://openmrs.staging.openconceptlab.org/login');
+});
+
+Given('the user goes to the openmrs application', () => {
+  cy.visit('http://localhost:8080/openmrs/login.htm');
+});
+
+When('the user logs into the Dictionary Manager', () => {
+  cy.get("#username").type('openmrs');
+  cy.get("#password").type('Openmrs123');
```

Comment:
It would be better if we could get these via environment variables... then we could leverage GitHub secrets and not need to have credentials embedded everywhere (it also would make the tests more flexible).

Reply:
@ibacher if we want to use github secrets especially for passwords, then someone with permissions needs to add them to the secrets of this repository, I can add the usernames to the `env` in `cypress.json` and then add passwords to secrets. OR should I do like what we do on Dictionary Manager something like `export const getUser = () => Cypress.env("USERNAME") || "openmrs";`

Reply:
Done... I've created the secrets `OCL_USER_NAME`, `OCL_USER_PASSWORD`, `OPENMRS_USER_NAME` and `OPENMRS_USER_PASSWORD` for this repo.

Reply:
Thanks

Reply:
@ibacher can you also add the token to the secrets, please.